### DISCUSSION TITLE
airflow cli group commands does not work after upgraded

### DISCUSSION THREAD

**Discussion by bennfocus** [apache/airflow#17309] [Q&A] [answered]
Hi, 

We have met a problem after upgrade airflow to newest version 2.1.2, and I am asking help here.

Our airflow was running in docker, we have our own docker image which based on ubuntu 18.04 and python 3.7

After upgrading, some group commands shows this (I have tried ubuntu 20.04 with python 3.8, get same issue):

```shell
root@72812b6c2f19:/airflow# airflow dags list
usage: airflow [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: airflow --help [cmd1 cmd2 ...]
   or: airflow --help-commands
   or: airflow cmd --help

error: invalid command 'dags'
```

but some are working:
```shell
root@72812b6c2f19:/airflow# airflow users list
id | username | email               | first_name | last_name | roles
===+==========+=====================+============+===========+======
1  | test  | test@test.com | test        | data      | Admin
```

**Comment by bennfocus** [comment, marked as answer]
We found this issue is because one of our Dag modules has a setup.py inside, and it didn't check __name__ == '__main__'.

And setup doesn't support group commands, so that's why it worked before not now.

Hope this answer helps other too.

**Reply by potiuk** [reply]
Ough.

**Reply by leahecole** [reply]
Ooof - glad you figured it out and thank you for posting the answer here for others to see!
